src/MSBuild/XMake.cs

```diff
@@ -3444,7 +3444,7 @@ private static bool DoesEnvironmentSupportLiveLogger()
             }
             // If terminal is dumb
             if (
-                (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && Environment.GetEnvironmentVariable("WT_SESSION") == "")
+                (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && (Environment.GetEnvironmentVariable("WT_SESSION") == "" || Environment.GetEnvironmentVariable("WT_SESSION") == null))
```

Comment:
nit:
```suggestion
                (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && string.IsNullOrEmpty(Environment.GetEnvironmentVariable("WT_SESSION")))
```

I find this simpler and often use it even if I think only one of those will show up just because we often realize we need both, and they're often intended to indicate the same thing 🙂